week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,11 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var pairsByIndex = pairsByIndexRaw.filter(function(pairs){
+   if (Array.isArray(pairs) && pairs.length==2){
```

Comment:
Really nice work using `Array.isArray`! That's one of the JavaScript weirdnesses that I talked about last week 🙂